tests/hpc/gnu_compilers_hpc.pm

```diff
@@ -25,13 +25,10 @@ sub run ($self) {
     assert_script_run qq{module av | grep gnu/$version};
     assert_script_run 'module load gnu';
     assert_script_run qq{env |grep "MODULEPATH=/usr/share/lmod/moduledeps/gnu-$version"};
-    if (zypper_call("in gnu$version-compilers-hpc-devel", exitcode => [107]) == 107) {
-        record_soft_failure 'bsc#1212351 Type in posttrans script for the non-base Compiler Version cause Script to fail';
-    }
+
```

Comment:
How will be installation of devel package tested now? I see that soft fail was removed, but installation as well.

I see `zypper -n in gnu12-compilers-hpc gnu12-compilers-hpc-devel` in updated VR https://aquarius.suse.cz/tests/19273#.

But I don't see installation od devel package in the latest commit at all.

Reply:
I asked to move that to a separate git commit to make sure commits are atomic.
Check this please:
 https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17913#discussion_r1346903724

Reply:
in other words all I wanted is to have 2 commits; one removing the soft-fail and 2nd doing zypper in -devel

Reply:
Sorry. I understood that you ask for a second commit. i just thought i could do that in another PR to fasten things. I was waiting this to be merged first tho.